### PR TITLE
fixes URLs in CV builder to be latex-safe

### PR DESCRIPTION
**Pull request by scopatz**
and some black changes.  CC @CJ-Wright

### CODE REVIEW

**Comment by scopatz** [comment]
That didn't seem to work

File: tests/outputs/cv/scopatz.tex [hunk 1]
```diff
@@ -327,29 +327,29 @@
...
-    { 
+    {
         \resitem{This course covers conventional reactors.}
-      
-      
+
+
         \resitem{
-            \href{https://docs.google.com/document/d/1uMAx_KFZK9ugYyF6wWtLLWgITVhaTBkAf8-PxiboYdM/edit?usp=sharing}{Syllabus}.
+            \href{https://docs.google.com/document/d/1uMAx\_KFZK9ugYyF6wWtLLWgITVhaTBkAf8-PxiboYdM/edit?usp=sharing}{Syllabus}.
```
Comment (CJ-Wright) [inline tests/outputs/cv/scopatz.tex]: Did you mean to put in a `\` before the `_`?

Comment (scopatz) [reply tests/outputs/cv/scopatz.tex]: Yep, absolutely.  `_` on their own aren't renderable.

**Comment by scopatz** [comment]
It seems like the trailing whitespace is being kept in for some reason.

File: tests/outputs/cv/scopatz.tex (hunk 1 above)
Comment (CJ-Wright) [reply tests/outputs/cv/scopatz.tex]: Yes but `\href` doesn't render the URL, right? It should give a link named `Syllabus` and the link is the URL.

Comment (scopatz) [reply tests/outputs/cv/scopatz.tex]: Maybe it is just `#` that fail to render without and escape then.

**Comment by scopatz** [comment]
Alright @CJ-Wright - I believe that this is all fixed up.

**Pull request merged**